de_web_plugin.cpp

```diff
@@ -5122,6 +5116,7 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const deCONZ::
                             manufacturer.endsWith(QLatin1String("fvq6avy")) ||
                             manufacturer.endsWith(QLatin1String("w7cahqs")) ||
                             manufacturer.endsWith(QLatin1String("wdxldoj")) ||
+                            manufacturer.endsWith(QLatin1String("hn3negr")) ||
                             manufacturer.endsWith(QLatin1String("oclfnxz")) ||
                             manufacturer.endsWith(QLatin1String("GbxAXL2")))
```

Comment:
IDK how to do in this part, the node is not created, have just a QString with the manufacture name, or need to create a new fonction that use only the manufacture name instead of the node ?

Reply:
Ah ok I thought the lightNode already exists here (haven't looked closely).

Yes we need more helper functions, they can be added in future PR's.